.github/workflows/android.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
     - master
     tags:
-    - '*'
+    - '**'
```

Comment:
What does this change do>

Reply:
Makes sure that tags in the form of `orbit2/1.0.0` are handled. The single star glob does not match slashes